src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -85,7 +87,17 @@ std::shared_ptr<ov::Model> add_slices_to_kvcache_inputs(const std::shared_ptr<ov
 
 void reshape_to_static(std::shared_ptr<ov::Model> model,
                        const uint32_t input_size,
-                       const uint32_t kvcache_size) {
+                       const uint32_t kvcache_size,
+                       const std::filesystem::path& path) {
+    auto config_file_path = path / "config.json";
```

Comment:
This shouldn't be part of `reshape_to_static` function as it will be also used for blobs for which this function isn't relevant.

Please isolate these changes to separate function(s), e.g: 
```
struct KVAxesPosition {
    uin32_t batch;
    uin32_t seq_len;
};

KVAxesPosition get_kv_axes(const std::string& model_type) { ... }
KVAxesPosition read_kv_axes_from(const std::string& filepath) { ... };
```

Reply:
Ok, done. I decided to create a "get_model_type" function instead since the kvcache dims need to be identified anyway here: https://github.com/openvinotoolkit/openvino.genai/blob/master/src/cpp/src/llm_pipeline_static.cpp#L223